config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  root 'application#hello'
+  root 'login#login'
+  get '/home', to: 'home#home'
+  get '/register', to: 'users#new'
+  post '/register',  to: 'users#create'
+  get '/login', to: 'login#login'
```

Comment:
The rails-y way of doing things, as far as I know, is to create empty models in the `new` controller action so that way the associated forms can populate any model dependent fields. Those models are then regenerated before the `create` controller action.